File: tests/fixtures/sample_cs/calculator.cs
using System;
using System.Collections.Generic;

namespace Calculator.App
{
    public interface ICalculator
    {
        int Add(int n);
        int GetValue();
    }

    public class Calculator : ICalculator
    {
        private int _value;
        private string _name;

        public Calculator(int initial, string name)
        {
            _value = initial;
            _name = name;
        }

        public int Add(int n)
        {
            _value += n;
            Console.WriteLine($"Adding {n}, result: {_value}");
            return _value;
        }

        public int Subtract(int n)
        {
            _value -= n;
            return _value;
        }

        public int GetValue()
        {
            return _value;
        }

        public string GetName()
        {
            return _name;
        }
    }

    public enum OperationType
    {
        Addition,
        Subtraction,
        Multiplication,
        Division
    }
}

File: tests/fixtures/sample_swift/calculator.swift
import Foundation

protocol Calculatable {
    func add(_ n: Int) -> Int
    func subtract(_ n: Int) -> Int
    func result() -> Int
}

class Calculator: Calculatable {
    private var value: Int
    private let name: String

    init(initial: Int = 0, name: String = "default") {
        self.value = initial
        self.name = name
    }

    func add(_ n: Int) -> Int {
        value += n
        print("Adding \(n), result: \(value)")
        return value
    }

    func subtract(_ n: Int) -> Int {
        value -= n
        return value
    }

    func result() -> Int {
        return value
    }

    func reset() {
        value = 0
    }
}

enum Operation: String, CaseIterable {
    case addition = "Addition"
    case subtraction = "Subtraction"
    case multiplication = "Multiplication"
    case division = "Division"

    func description() -> String {
        return "Operation: \(rawValue)"
    }
}

struct CalculationResult {
    let operation: Operation
    let input: Int
    let output: Int
    let timestamp: Date

    func formatted() -> String {
        return "\(operation.rawValue): \(input) -> \(output)"
    }
}
